src/test/java/org/codeseoul/event_member_management/member/MemberRestIntegrationTests.java

```diff
@@ -64,9 +61,6 @@ private Member createAnotherMember() {
     public void findsAllMembers() throws Exception {
         Member anotherMember = createAnotherMember();
         memberRepository.saveAndFlush(anotherMember);
-        List<Member> members = new ArrayList<>();
-        members.add(aMember);
-        members.add(anotherMember);
```

Comment:
I should have caught this during last review. I'll change my review style. I'll also look into adding a linter

Reply:
Originally, I was using VS code, which doesn't have many warnings for redundant code, but I switched back to IntelliJ and found a whole bunch of stuff.